messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/reactive/streams/KafkaGratefulShutdownIT.java

```diff
@@ -48,6 +49,14 @@ public class KafkaGratefulShutdownIT {
 
     @Order(1)
     @Test
+    public void testConnection() throws InterruptedException {
+        Thread.sleep(5000); // TODO:  https://github.com/quarkusio/quarkus/issues/41441
```

Comment:
Waiting 5 s is bit of a blow. Please at the very least adjust comment based on the https://github.com/quarkusio/quarkus/issues/41441 feedback.

Reply:
hmm, if you plan to do this in the future and just want to keep TODO, ok.

Reply:
Add topic creation and lowered it to 1 second